examples/web-wagmi/src/profiles/UseCreateProfile.tsx

```diff
@@ -1,9 +1,9 @@
 import { useCreateProfile, useProfile } from '@lens-protocol/react';
 import { useState } from 'react';
 
+import { ProfileCard } from './components/ProfileCard';
```

Comment:
as discussed, personally, I prefer the order of most far away -> closer

```typescript
import external from 'external'
import { Something } from '../something'
import { SomethingElse } from './something'
```

Reply:
I reverted update of `eslint-plugin-import` so the order is not changed